report/lambda_function.py

```diff
@@ -511,7 +511,7 @@ def create_report(df_releases: DataFrame, dashboard_url: str) -> None:
     </html>
     '''
 
-    with open("test.html", "w", encoding='utf-8') as file:
+    with open("/tmp/test.html", "w", encoding='utf-8') as file:
         file.write(template)
```

Comment:
Did you need this `.html` file? I only added it for testing purposes and forgot to remove. I will remove it from next revision if you don't need it. The email report does not require it to work.

Reply:
Yeah I don't need it, good point